test/ImportCommand.js

```diff
@@ -84,10 +84,10 @@ describe("ImportCommand", () => {
       expect(await lastCommitInDir(externalDir)).toBe("Branch merged");
 
       await lernaImport(externalDir, "--flatten");
-      expect(await lastCommitInDir(testDir)).toBe("Branch merged");
+      expect(await lastCommitInDir(externalDir)).toBe("Branch merged");
 
-      const newFilePath = path.join(testDir, "packages", path.basename(externalDir), conflictedFileName);
-      expect(await pathExists(newFilePath)).toBe(true);
+      const newFilePath = path.join(externalDir, conflictedFileName);
```

Comment:
This is definitely not correct. We're testing that the conflicted file was imported into the target repo, here.

Reply:
What method is supposed to import that conflicted file?